beacon-chain/core/state/epoch_processing_test.go

```diff
@@ -0,0 +1,105 @@
+package state
+
+import (
+	"bytes"
+	"testing"
+
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+func TestEpochAttestations_ok(t *testing.T) {
+	if params.BeaconConfig().EpochLength != 64 {
+		t.Errorf("EpochLength should be 64 for these tests to pass")
```

Comment:
This should fatal

Reply:
I disagree, if the test continues and passes then you have even bigger problems. Change to Errorf please